packages/framework/pr_tools/trilinosprhelpers/TrilinosPRConfigurationStandard.py

```diff
@@ -83,7 +83,7 @@ def execute_test(self):
             gpu_indices = gpu_utils.list_nvidia_gpus()
             self.message(f"-- REMARK: Using {slots_per_gpu} slots per GPU")
             self.message(f"-- REMARK: Using GPUs {gpu_indices}")
-            cmd.append(f"-DEXTRA_CONFIGURE_ARGS:STRING=\"-DTrilinos_AUTOGENERATE_TEST_RESOURCE_FILE:BOOL=ON;-DTrilinos_CUDA_NUM_GPUS:STRING={len(gpu_indices)};-DTrilinos_CUDA_SLOTS_PER_GPU:STRING={slots_per_gpu}\"")
+            cmd.append(f"-DEXTRA_CONFIGURE_ARGS:STRING=-DTrilinos_AUTOGENERATE_TEST_RESOURCE_FILE:BOOL=ON; -DTrilinos_CUDA_NUM_GPUS:STRING={len(gpu_indices)}; -DTrilinos_CUDA_SLOTS_PER_GPU:STRING={slots_per_gpu}")
```

Comment:
Excellent call.  I missed that `subprocess` is not being used in shell mode, which means that the list gets passed directly to it instead of concatenated into a string, and it handles the different arguments itself with no need for quoting.